tests/mocks/microsalt_analysis_mock.py

```diff
@@ -0,0 +1,17 @@
+import logging
+from datetime import datetime
+
+from cg.meta.workflow.microsalt import MicrosaltAnalysisAPI
+from pathlib import Path
+
+LOG = logging.getLogger(__name__)
+
+
+class MockMicrosaltAnalysis(MicrosaltAnalysisAPI):
```

Comment:
Do we need this or can you just mock it like similar to what you did here:
```
  mocker.patch.object(TrailblazerAPI, "is_latest_analysis_ongoing")
    TrailblazerAPI.is_latest_analysis_ongoing.return_value = False


```

Reply:
Smart, fixed!